src/test/java/hw2/base/BaseTest.java

```diff
@@ -0,0 +1,155 @@
+package hw2.base;
+
+import hw2.enums.LoginUser;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import java.util.List;
+
+import static hw2.base.Const.*;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public abstract class BaseTest {
+
+    protected WebDriver driver;
+
+    @BeforeClass
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver", "src\\test\\resources\\chromeDriver\\chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+    }
+
+    @AfterClass
+    public void tearDown() {
+        driver.quit();
+    }
+
+    //ex1 and ex2 common steps
+    public void openURL() {
```

Comment:
This could mislead... which url?
I should understand that we open main page. Rename pls